src/common/visualization-manifests/line-chart/line-chart.ts

```diff
@@ -90,13 +90,27 @@ const rulesEvaluator = visualizationDependentEvaluatorBuilder
 
     const colorSplit = splits.getSplit(1);
     const colorDimension = findDimensionByName(dataCube.dimensions, colorSplit.reference);
-
     const newColorSplit = adjustColorSplit(colorSplit, colorDimension, series, appSettings.customization.visualizationColors);
 
     return Resolve.automatic(8, {
       splits: new Splits({ splits: List([newColorSplit, newTimeSplit]) })
     });
   })
+  .when(Predicates.areExactSplitKinds("number", "*"))
+  .then(({ splits, series, dataCube, appSettings }) => {
```

Comment:
Same logic as in `then` from line 85. With small adjustments between time and number split types.

Maybe we could extract some common code but i think it's futile. Let's push forward refactor of visualisation controls and remove resolvers altogether.